Assets/UI/Panels/unitpanel.lua

```diff
@@ -918,7 +922,8 @@ function View(data)
   end
 
   -- Populate Earned Promotions UI
-  if (not UILens.IsLensActive("Religion") and data.Combat > 0 and (UI.GetInterfaceMode() ~= InterfaceModeTypes.CITY_RANGE_ATTACK) and (UI.GetInterfaceMode() ~= InterfaceModeTypes.DISTRICT_RANGE_ATTACK)) then
+  -- QCUI modified to not allow promotion when in city attack interface (?)
```

Comment:
@Azurency This condition seems to have been modified by CQUI to exclude certain InterfaceModTypes. Please decide whether to keep this way or not.